client/reader/stream/featured-asset.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+export default class FeaturedAsset extends React.Component {
```

Comment:
Ok, I wired it into the stream here https://github.com/Automattic/wp-calypso/pull/8230/commits/b07f744093bd2ae3e3ba080da73f54f3bea07c02